src/pypi_attestations/_impl.py

```diff
@@ -331,3 +331,54 @@ def _ultranormalize_dist_filename(dist: str) -> str:
         return f"{name}-{ver}.tar.gz"
     else:
         raise ValueError(f"unknown distribution format: {dist}")
+
+
+class Publisher(BaseModel):
```

Comment:
I _think_ we'll want to do this as a discriminated enum instead, using Pydantic's support for that. But that's a larger change, so IMO we can iterate on that in a subsequent PR.

Reply:
Opened #38 for this -- that'll get merged into this PR 🙂